src/_posts/languages/php/2000-01-01-laravel.md

```diff
@@ -279,73 +302,42 @@ class SchedulerDaemon extends Command
 }
 ```
 
-Don't forget to add the following line to the `Procfile` of your application:
+In the example above, you can see that the defined scheduler runs every minute.
+(If you look closely, you will also see that our scheduler makes use of the one
+provided by Laravel, `schedule:run`.)
+
+You now have to tell Scalingo how to start this new scheduler. To do this,
```

Comment:
I have the feeling that "tell" (https://www.wordreference.com/enfr/tell) is not exactly the word to use 🤔 

```suggestion
You now have to instruct Scalingo how to start this new scheduler. To do this,
```